trillian/ctfe/ct_server/main.go

```diff
@@ -121,23 +121,27 @@ func main() {
 			glog.Infof("Removing our presence in %v with %+v", *etcdMetricsService, byeMetrics)
 			etcdRes.Update(ctx, *etcdMetricsService, byeMetrics)
 		}()
-	} else {
+		dialOpts = append(dialOpts, grpc.WithBalancer(grpc.RoundRobin(etcdRes)))
```

Comment:
I think this line would be better up by where `etcdRes` is defined

Reply:
Done